src/components/Collapsible/Collapsible.tsx

```diff
@@ -66,20 +64,24 @@ const Collapsible: React.FC<CollapsibleProps> = ({
 
   return (
     <Container className={className} withBackground={isOpen}>
-      <Header alignItems="center" onClick={handleHeaderClick}>
-        <StyledIcon
-          isRotated={isOpen}
-          name={SSCIconNames.chevronRight}
-          type={IconTypes.ssc}
-        />
-        <div>
-          <Text size={TextSizes.md}>{title}</Text>
-          <Subject>{subject}</Subject>
-        </div>
+      <Header paddingSize="lg" paddingType="squish" onClick={handleHeaderClick}>
```

Comment:
Please replace "lg" with `{SpaceSizes.lg}` and "squish" with `{PaddingTypes.squish}`

Reply:
is there a reason to use constants as opposed to the string literal?
We seem to be using both variants in the different components

Reply:
well, It seems to be a way to standardize code. Also, It's intended to be a way to prevent typo errors as well. @ajkl2533 should we run a search and destroy for those literals?

Reply:
@dclerici-ssc we can, I hope there are not many strings in the DS tho